README.md

```diff
@@ -1 +1,141 @@
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖  Portfolio-project <a name=" Portfolio-project">My Portfolio Project</a>
+
+
+
+**Portfolio-project** This is my portfolio project. 
+Portfolio-project is a mobile and desktop responsive page.
+
+## 🛠 Built With HTML <a name="For Text"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML5 and CSS which are the latest versions were used in building this project.
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">JAVASCRIPT</a></li>
+  </ul>
+</details>
+
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **[key_feature_1]** Portfolio-project built with HTML and CSS.
+- **[key_feature_2]** Responsive mobile design portfolio.
+
+
+- **feature_HOME-PAGE**
+
+## 🚀 Live Demo <a name="Live Demo"></a>
+
+
+)
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Get your pc.
+Clone this project to your local machine.
+
+To get a local copy up and running, follow these steps.
+
+
+
+### Prerequisites
+Basic knownledge of computer.
+
+In order to run this project you need:
+Live server.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  cd Microverse_portfolio
+  git clone https://github.com/adeola003/Microverse_portfolio.git
+```
+
+## 👥 Authors <a name="Abdel-Khafid SALAOU"></a>
+
+
+👤 **Abdel-Khafid**
+
+<a href="https://github.com/adeola003">- GitHub: [@githubhandle]</a>
+<a href="https://twitter.com/khadaf6">- Twitter: [@twitterhandle]</a>
+<a href="https://www.linkedin.com/in/abdel-khafid-salaou-a18396161/">- LinkedIn: [LinkedIn]</a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- **About-page**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+If you like this project you can follow my github or star it
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+All my 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is <a href="https://github.com/adeola003/Microverse_portfolio/blob/3ca5489580c13cb372bd064c1f13825c2d17ba40/LICENSE.md">MIT licensed</a>.
```

Comment:
Your MIT license is not properly linked. Kindly download the [template ](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository)available on GitHub and link it like this 👇 
This project is [MIT](./LICENSE) licensed.

Reply:
Okay, Madam @Lizdev-05. So if I understand you well, I just have to remove "licensed" from the "a" element and put it just after the closing tag. is that right?
Thank you for your time @Lizdev-05.

Reply:
@Lizdev-05 I implemented all the changes I will be waiting for your confirmation to submit it as this is my last authorized review request.